src/core/math/vec3/random.js

```diff
@@ -1,9 +1,9 @@
-module.exports = randomNonParallel
+module.exports = random
 
 const normalize = require('./normalize')
 const fromValues = require('./fromValues')
 
-function randomNonParallel (input) {
+function random (input) {
```

Comment:
same remarks as in the index
I think it should be called 'createNonParallel' or something ?

Reply:
Well... dot and cross return a new vector, but those are not named...

createCross()
createDot()

Im flexible but we should be consistent.

Reply:
ok, so err 'nonParallel' ?